source/stylesheets/base/elements/typography.styl

```diff
@@ -64,6 +70,28 @@ a:active {
     text-decoration: underline;
 }
 
+@media print {
+    a {
+        @extends $print-plain;
+
+        &[href^='http'] { // print links on page for off site links
+            text-decoration: underline !important;
+            word-wrap: break-word;
```

Comment:
Hm. That's a problem because that font face doesn't actually have a lower case character set >_< I can use a different font for the a[href] stuff though. Better a miss matched font than a broken link. Also, then the links aren't all yelling at us :P Good catch.